BrowserKit/Sources/Common/Theming/DarkTheme.swift

```diff
@@ -30,6 +30,7 @@ private struct DarkColourPalette: ThemeColourPalette {
     var layerLightGrey30: UIColor = FXColors.LightGrey30
     var layerSepia: UIColor = FXColors.Orange05
     var layerConfirmation: UIColor = FXColors.Green80
+    var layerWarning: UIColor = FXColors.Yellow70.withAlphaComponent(0.77)
```

Comment:
I think we should add the new colors to the design system and use them here instead.

Reply:
<img width="346" alt="Screenshot 2023-09-08 at 15 59 31" src="https://github.com/mozilla-mobile/firefox-ios/assets/26011662/c8d45735-05fd-47eb-96bd-0cb8e02192f7">
I'm not sure what you mean. This is the color named yellow70 with an alpha component of 0.77.

Reply:
The color name is right there in your screenshot: `fxColorYellow70A77`. 

Maybe that's a question for @lmarceau : should we add colors that just differ in alpha from an existing color but have a distinct name in the design system?

Reply:
I see now. So the question is if we want an additional color  `fxColorYellow70A77`.
Tagging @yoanarios as well since I know she did some theming work.

Reply:
So far we added mobile themed colors with alpha on them. I can bring that as a question to the designers. But I wouldn't block this PR as we did this on other colors as well, so if we change how we do this we'll change this in another PR.